xena_gdc_etl/xena_dataset.py

```
@@ -1459,170 +1472,222 @@ def transform(self):
 
         message = 'Make Xena matrix for {} data of {}.'
         print(message.format(self.xena_dtype, self.projects))
-        total = len(self.raw_data_list)
-        count = 0
-        bio_dfs = []
-        clin_dfs = []
-        for path in self.raw_data_list:
-            count = count + 1
-            print('\rProcessing {}/{} file...'.format(count, total), end='')
-            sys.stdout.flush()
-            # `read_biospecimen` and `read_clinical` will check file format
-            try:
-                df = read_clinical(path)
-                clin_dfs.append(df)
-            except Exception:
-                try:
-                    df = read_biospecimen(path)
-                    bio_dfs.append(df)
-                except Exception:
-                    raise TypeError('Fail to process file {}.'.format(path))
-        print('\rAll {} files have been processed. '.format(total))
-        try:
-            bio_matrix = (
-                pd.concat(bio_dfs, axis=0)
-                .replace(r'\r\n', ' ', regex=True)
-                .replace(r'^\s*$', np.nan, regex=True)
-                .dropna(axis=1, how='all')
-            )
-        except Exception:
-            bio_matrix = pd.DataFrame()
-        try:
-            clin_matrix = (
-                pd.concat(clin_dfs, axis=0)
-                .replace(r'\r\n', ' ', regex=True)
-                .replace(r'^\s*$', np.nan, regex=True)
-                .dropna(axis=1, how='all')
-            )
-        except Exception:
-            clin_matrix = pd.DataFrame()
```

Both `try` would fail and gives empty `bio_matrix` and `clin_matrix` if nothing is done. I think it's fine to let them fail but I would also agree if you think it's not a good or readable logic for the future. Overall, this is a patten following the `download` and the `for` loop above. That's why I'm OK running it. Feel free to change it if you want.